xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -392,6 +392,22 @@
     },
     "additionalProperties": { "type": [ "null", { "$ref": "Global.String.NotEmpty", "required": true } ] }
   },
+  "Video.Rating": {
+    "type": "object",
+    "properties": {
+      "rating": { "type": "number", "required": true },
+      "votes": { "type": "integer" },
+      "default": { "type": "boolean" }
+    }
+  },
+  "Video.Ratings": {
+    "type": "object",
+    "additionalProperties": { "type": "$ref": "Video.Rating" }
+  },
+  "Video.Ratings.Set": {
+    "type": "object",
+    "additionalProperties": { "type": [ "null", { "$ref": "Video.Rating", "required": true } ] }
```

Comment:
In that case, I don't see where the `Video.Ratings` (plural) type is used or referenced by a field so it appears (with my limited understanding of this JSON syntax) to be redundant/unnecessary as only `Video.Rating` is used by `Video.Ratings.Set`, which in turn is used only by the `ratings` field/property.